chart/values.schema.json

```diff
@@ -521,7 +521,8 @@
                 "LocalKubernetesExecutor",
                 "CeleryExecutor",
                 "KubernetesExecutor",
-                "CeleryKubernetesExecutor"
+                "CeleryKubernetesExecutor",
+                "airflow.providers.amazon.aws.executors.ecs.ecs_executor.AwsEcsExecutor"
```

Comment:
I'm curious why do we need allowed list at all?
Users can also roll thier own executors.
I think maybe this limitation is prior to AIP-51?
cc @jedcunningham

Reply:
> I'm curious why do we need allowed list at all?

+1 for removing it **if possible**

Reply:
Hi all, I'm wondering if you have discussed this internally?

Reply:
Given the situation I think we better unblock first and then see how we can refactor/fix.
So probably we should accept this PR and open a followup issue cc @o-nikolas

Reply:
Somehow I missed the first time I was tagged in this thread.

> @o-nikolas, is there a plan to eventually have a short alias for AwsEcsExecutor vs requiring the full path?

A while ago I had suggested a way to "register" custom executors with the same sort of alias namespace that the Airflow core executors use. But it didn't get much traction.

We could probably do some python trickery to make the aws executors importable from something like `airflow.providers.amazon.aws.executors` using `__init__` or another module in that dir that we import all the available executors into? Or maybe one module per executor?

> So probably we should accept this PR and open a followup issue cc @o-nikolas
I'm not so sure of this. Can someone clarify if the 50 char limit is a blocker for this PR? I see that the value is getting truncated which I think would completely break the import path, right? So as is, this PR is not mergeable?

Reply:
I think a reasonable fix to unblock this would be to import the `AwsEcsExecutor` class inside the `__init__.py` module of the `ecs` directory. So that you could import it as:

`airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor`

This is under 63 characters (but above 50, but I'm confused why we're limiting it to 50 since the documented limit for labels seems to be 63 maybe @jedcunningham and @LipuFei can comment on that)

Reply:
I've changed the allowed values according to #39093